src/Tizen.NUI/src/internal/Interop/Interop.DefaultWindowData.cs

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.Runtime.InteropServices;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class DefaultWindowData
+        {
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_New")]
+            public static extern global::System.IntPtr New();
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_delete_DefaultWindowData")]
+            public static extern void DeleteDefaultWindowData(HandleRef nuiDefaultWindowData);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_Assign")]
+            public static extern void Assign(HandleRef nuiDefaultWindowData1, HandleRef nuiDefaultWindowData2);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_SetPositionSize")]
+            public static extern void SetPositionSize(HandleRef nuiDefaultWindowData, HandleRef nuiPositionSize);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_GetPositionSize")]
+            public static extern global::System.IntPtr GetPositionSize(HandleRef nuiDefaultWindowData);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_SetWindowType")]
+            public static extern void SetWindowType(HandleRef nuiDefaultWindowData, int nuiWindowType);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_GetWindowType")]
+            public static extern int GetWindowType(HandleRef nuiDefaultWindowData);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_SetTransparency")]
+            public static extern void SetTransparency(HandleRef nuiDefaultWindowData, bool nuiTransparency);
+
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_GetTransparency")]
```

Comment:
How about marshal boolean type to 1byte?
Your current code has no problem but I'd like to recommand you to use 1byte data type for `bool` for consistency with other NUI code. (It's 4 bytes by default. [MS Guide](https://learn.microsoft.com/en-us/dotnet/api/system.boolean?view=net-7.0#booleans-and-interop))
```suggestion
            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_DefaultWindowData_GetTransparency")]
            [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]
```
This also needs to update your csharp binder patch to return `bool` instead of `unsinged int`.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/290165/
```
SWIGEXPORT bool SWIGSTDCALL CSharp_Dali_DefaultWindowData_GetTransparency(void* nuiDefaultWindowData)
```